pyk/src/pyk/cli_utils.py

```diff
@@ -51,8 +51,12 @@ def run_process(
     try:
         res = subprocess.run(args, input=input, env=env, capture_output=True, check=True, text=True)
     except CalledProcessError as err:
+        sys.stderr.write(err.stderr)
+        sys.stderr.flush()
         logger.log(log_level, f'Completed with status {err.returncode}: {command}')
         raise
+    sys.stderr.write(res.stderr)
+    sys.stderr.flush()
```

Comment:
I don't think this is a good idea in general. In certain cases I guess it is, but then the caller can do the printing.

But we can also add a boolean argument with a default value to the function to enable this feature when needed.

Reply:
How could this be a bad idea though? If a process we are calling produces stderr, we want to be able to see that. I can put it behind a flag, but I think it should be turned on by default, it's harder to track down where an error is coming from if this isn't output.

Reply:
I added a flag for disabling it if the process succeeds. If we notice it becomes really annoying for something, we can change it again later.

The motivation for this is that the ksummarizer was generating rules for a long time with incorrect variable naming conventions, but I didn't know at all because stderr of the prover was being dropped silently (I had no hint things were wrong). These were warnings I wanted to fix directly, not ignore.

Reply:
I think using `subprocess.check_output(stderr=None, ...)` would get you this behaviour, with the subprocess printing immediately rather than upon completion. We can add `stderr` as an optional parameter, to allow other behaviours.

Reply:
I've backed out this change completely, can deal with it another time.